Improve vulnerability detector section (#7243)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/vuln-detector.rst

vulnerability-detection
=======================

.. note::

>    This section applies for 4.8.0 and later versions of the Wazuh. If you are using an older version, this section is called `vulnerability-detector`.

Reviewer: ```suggestion
   This section applies to 4.8.0 and later versions of Wazuh. If you use an older version, this section is called `vulnerability-detector`.
```

Author: Thanks!
